R/mod_review_data_fct_helpers.R

```diff
@@ -88,29 +88,33 @@ update_review_data <- function(
   # - the data point will be reviewed. a new row with the same item and edit date-time 
   #   is added, with new review status and new timestamp
   
-  df <- latest_review_data |> 
-    # including event_date in join by should work, even for Adverse events, since 
-    # if event_date changes, the edit date-time also changes, and thus all rows 
-    # will be taken into consideration.
-    dplyr::full_join(review_df, by = names(latest_review_data)) |> 
-    add_missing_columns(c("timestamp", "reviewed", "comment", "status"))
-  
-  df <- df |> 
-    tidyr::replace_na(list(timestamp = update_time, reviewed = "No", comment = "")) |> 
-    # now add status labels to the new rows:
+  # Grab the records not present in the review db table. Impute missing values.
+  new_records <- latest_review_data |> 
+    dplyr::anti_join(review_df, by = common_vars) |> 
+    add_missing_columns(c("timestamp", "reviewed", "comment", "status")) |> 
```

Comment:
I don't think add_missing_columns is needed if we are writing it this way, since the `mutate` call will add these columns anyways.